cmd/kubeadm/app/componentconfigs/kubelet.go

```diff
@@ -233,60 +230,6 @@ func (kc *kubeletConfig) Default(cfg *kubeadmapi.ClusterConfiguration, _ *kubead
 	}
 }
 
-// Mutate modifies absolute path fields in the KubeletConfiguration to be Windows compatible absolute paths.
-func (kc *kubeletConfig) Mutate() error {
-	// TODO: use build tags and move the Windows related logic to _windows.go files
-	// once the kubeadm code base is unit tested for Windows as part of CI - "GOOS=windows go test ...".
```

Comment:
as the TODO explains we cannot do this until we start running unit tests for GOOS=windows.
if we move it now the unit tests will be skipped because the k8s CI only runs on linux.

Reply:
👌, Thanks for review,  will Kubeadm become a separate kubernets project？
If so, I will close this PR. 😄